drivers/lora/Kconfig.semtech

```diff
@@ -25,6 +25,12 @@ config LORA_SX1276
 	help
 	  Enable LoRa driver for Semtech SX1276.
 
+config LORA_SX126X
+	bool "Semtech SX126x driver"
+	select HAS_SEMTECH_SX126X
```

Comment:
I wonder what is the use of HAS_SEMTECH_SX126X. It is selected, but not used anywhere.

Reply:
It's used in the LoRaMAC-node repo to include the SX126X driver in the build. See https://github.com/zephyrproject-rtos/loramac-node/pull/4.